### PR TITLE
Customize the admin site

### PR DESCRIPTION
**Pull request by pafff**
@bitrut

### CODE REVIEW

File: polls/models.py [hunk 1]
```diff
@@ -14,6 +14,9 @@ class Question(models.Model):
     def was_published_recently(self):
         now = timezone.now()
         return now - datetime.timedelta(days=1) <= self.pub_date <= now
+    was_published_recently.admin_order_field = 'pub_date'
```
Comment (bitrut) [inline polls/models.py:17]: Shouldn't it be `-pub_date`?

Comment (pafff) [inline polls/models.py:17]: @bitrut It was like that in the tutorial. I think it is just an example of how to enable sorting for fields that are not in database so the order shouldn't matter.

Comment (bitrut) [inline polls/models.py:17]: OK.

**Comment by bitrut** [comment]
:+1:

**Pull request merged**